pkg/oc/clusteradd/cmd.go

```diff
@@ -80,10 +80,9 @@ var availableComponents = map[string]func(ctx componentinstall.Context) componen
 	},
 }
 
-func NewCmdAdd(name, fullName string, out, errout io.Writer) *cobra.Command {
+func NewCmdAdd(name, fullName string, streams genericclioptions.IOStreams) *cobra.Command {
```

Comment:
nit: we could add a NewClusterAddConfig method in a future PR

Reply:
That's actually handled in https://github.com/openshift/origin/pull/20232